tests/testthat/test-test_dir.R

```diff
@@ -33,3 +33,27 @@ test_that('filter_test_scripts() with tricky names', {
     files[1]
   )
 })
+
+test_that('test_dir and `wrap`', {
+  ## forced to test screen output because tests outside of `testthat` blocks are
+  ## not recorded in return value, even when `wrap=TRUE` (see #596")
+
+  w.wrap <- capture.output(
+    test_dir('test_dir', filter='bare-expectations', wrap=TRUE)
```

Comment:
Sure, will address this evening.  I've tried to do it as much as possible (e.g. in the actual function definition), but old habits die hard.